docs/src/pages/[platform]/components/checkboxfield/react.mdx

```diff
@@ -156,6 +155,20 @@ import { CheckboxField } from '@aws-amplify/ui-react';
 
 ## CSS Styling
```

Comment:
Might as well decide on a consistent title here. CSS Styling is not accurate anymore IMO. I propose something like `Customization` or `Styling customization`